aws-lc-rs/Cargo.toml

```diff
@@ -2,6 +2,8 @@
 name = "aws-lc-rs"
 authors = ["AWS-LibCrypto"]
 version = "1.6.1"
+# this crate re-exports whatever sys crate that was selected
+links = "aws_lc_rs_1_6_1"
```

Comment:
Let's make this `aws_libcrypto_1_6_1`.  I don't want to confuse `aws_lc_rs` being a "native library".
"

Reply:
Yeah the only way to export build variables from this crate is by having a `links` attribute. I'm fine to call it whatever, though.

Reply:
How about `aws_lc_rs_1_6_1_libcrypto`?

Reply:
talked offline with @justsmth. we think `aws_lc_rs_1_6_1_sys` communicates best what's happening here.